lib/features/give/pages/home_page.dart

```diff
@@ -187,15 +187,19 @@ class _HomePageState extends State<HomePage> {
     BuildContext context,
   ) {
     final user = context.read<AuthCubit>().state.user;
+    final isUS = user.country == Country.us.countryCode;
```

Comment:
I was thinking to move this thing into the the Country enum extension as this is soo common in the entire codebase. So I am thinking of something like "Country.isUS" and you can just do Countr.fromCode(user.countryCode). And use the "country.isUS" whenever needed. What do you think?

Reply:
Like this? https://github.com/givtnl/givt-app/pull/498

Reply:
@Daniela510 There's a method in the Country enum you can just do "Country.fromCode(user.country).isUS"